lib/task-data/schemas/install-os-general.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
```

Comment:
@yyscamper Yeah, I see what you mean.  At a high level, I think a structure that has a set of definitions like `install/definitions/os-base` (for os-install-base), `install/definitions/os` (for os-install-general), and `install/definitions/centos` (for centos) might make the intent of these definitions a little clearer.  In any case, I'm fine with merging as is.